docs/source/getting-started/authenticators-users-basics.md

```diff
@@ -101,6 +101,9 @@ popular services:
 - Okpy
 - OpenShift
 
+NOTE: Open issue asking for more details on this generic implementation.
+It's not clear if this is a different implementation or if the JupyterHub OAuth
+_is_ the generic implementation.
```

Comment:
It's called `GenericOAuthenticator`, which is a generic configurable OAuth implementation. JupyterHub doesn't ship with support for external OAuth providers.

Reply:
Right above this list it says that JupyterHub's OAuthenticaator supports that list of services. Here you're saying that JupyterHub doesn't ship with support for external OAuth providers. Is it accurate to say that:

JupyterHub ships with a generic reference implementation, [GenericOAuthenticator](https://github.com/jupyterhub/oauthenticator/blob/master/oauthenticator/generic.py#L33) It is left up to the user to implement support for any OAuth provider that is not listed above.